packages/tofu/src/findGlobals.js

```diff
@@ -27,6 +38,12 @@ function findGlobals(ast) {
       if (nonReferenceIdentifiers.includes(parentType)) {
         return
       }
+
```

Comment:
also I really wanted to refactor this to use the builtin type guards in `@babel/traverse`'s `NodePath` objects.  unclear to me if it's any better--type prop comparison would probably be enough since it's likely a discriminated union--but it avoids us needing to do string comparisons on `.type` props (typo-prone).

e.g., `path.node.parent.type === 'VariableDeclarator'` would be rewritten as `path.parentPath.isVariableDeclarator()`